app/players/dwarves/dwarf_blacksmith.py

```diff
@@ -0,0 +1,19 @@
+from app.players.dwarves.dwarf import Dwarf
+
+
+class DwarfBlacksmith(Dwarf):
+    def __init__(self,
+                 nickname: str,
+                 favourite_dish: str,
+                 skill_level: int) -> None:
```

Comment:
It is a better practice to place each argument in the `__init__` on a new line to improve the readability. There should be no arguments on the same line with opening and closing parenthesis. Follow this practice of writing `__init__` in the future.